php/ext/google/protobuf/map.c

```diff
@@ -243,6 +243,18 @@ map_field_handlers->write_dimension = map_field_write_dimension;
 map_field_handlers->get_gc = map_field_get_gc;
 PHP_PROTO_INIT_CLASS_END
 
+void map_field_insure_created(const upb_fielddef *field,
```

Comment:
what does `insure` mean here?

Reply:
I mean, if not created, create it.
Should that be ensure?

Reply:
Yea that should be `ensure`.